File: Práctica 3/Coordenadas.java
//23900715N HERNANDEZ GOMEZ, JESSICA
public class Coordenadas {
	private int fila;
	private int columna;
	
	public Coordenadas(){
		fila = -1;
		columna = -1;
	}
	
	public Coordenadas(int i, int j){
		if(i>=0){
			fila = i;
		}else{
			fila = -1;
		}
		if(j>=0){
			columna = j;
		}
		else{
			columna = -1;
		}
	}

	public int getFila() {
		return fila;
	}

	public void setFila(int fila) {
		this.fila = fila;
	}

	public int getColumna() {
		return columna;
	}

	public void setColumna(int columna) {
		this.columna = columna;
	}
	
	public String toString(){
		String cadena = "(" + fila + "," + columna + ")" + "\n";
		return cadena;
	}

	public boolean equals(Object p){
		boolean devolver = false;

		if(p instanceof Coordenadas){
			if(((Coordenadas)p).getFila() == ((Coordenadas)p).getColumna()){
				devolver = true;
			}
		}

		return devolver;
	}
}

File: Práctica 2/Coordenadas.cc
//DNI 23900715N HERNANDEZ GOMEZ, JESSICA 
#include "Coordenadas.h"

std::ostream & operator<< (std::ostream &os, const Coordenadas &c) { //Funcion amiga de sobrecarga del operador salida
    os << "(" << c.fila << "," << c.columna << ")";
    return os;
}

Coordenadas::Coordenadas(){
    fila = -1;
    columna = -1;
}

Coordenadas::Coordenadas(int i, int j){
    if(i>=0 && j>=0){
        fila = i;
        columna = j;
    }else{
        fila = -1;
        columna = -1;
    }
}

Coordenadas::Coordenadas(const Coordenadas &c){
    fila = c.fila;
    columna = c.columna;
}

Coordenadas::~Coordenadas(){
    fila = -1;
    columna = -1;
}

Coordenadas & Coordenadas::operator=(const Coordenadas &c){
    fila = c.fila;
    columna = c.columna;
    return *this;
}

int Coordenadas::getFila(){
    return fila;
}

int Coordenadas::getColumna(){
    return columna;
}

void Coordenadas::setFila(int f){
    fila = f;
}

void Coordenadas::setColumna(int c){
    columna = c;
}